### PR TITLE
Added beoplay-software-update 1.0.0

### PR DESCRIPTION
**Pull request by bcomnes**
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/beoplay-software-update.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'beoplay-software-update' do
+  version '1.0.0'
+  sha256 '4d412e1d5119d0f6a4c4b6e19572320bb325348691d799bf5265b63bc0ac159a'
+
+  url 'http://www.beoplay.com/~/media/A6EA01EBD2F24D9CB021ADE273141B71'
+  name 'BeoPlay Software Update'
+  homepage 'http://www.beoplay.com/support/product%20support/beoplay%20updater'
+  license :closed
+
+  container nested: 'beoplay-product-updater_1.0.0.dmg'
```
Comment (vitorgalvao) [inline Casks/beoplay-software-update.rb]: `container nested: "beoplay-product-updater_#{version}.dmg"`

**Comment by vitorgalvao** [comment]
You’re only supposed to fill one of the parts. You’re not doing all three sections at once.

**Comment by bcomnes** [comment]
Whoops!  Just read and did the bullet points.  Correcting.

**Comment by vitorgalvao** [comment]
Thank you. Looks good.

**Pull request merged**